torch/ao/quantization/_pt2e/qat_utils.py

```diff
@@ -218,6 +218,15 @@ def _fuse_conv_bn_qat(m: GraphModule) -> GraphModule:
             # bias can be None
             if original_node is None:
                 continue
+            # We expect the subgraph rewriter to erase the non-literal args of the matched nodes.
```

Comment:
The problem is the `original_node` we want looks like this: `operator.getitem(None, 0)`, so there is no way to know whether the arg used to be a batchnorm node or not. The only thing we can check is if it's None.

Reply:
The check I have here is more general and guards against cases when a different "conv" is the input of the pattern as well for example. I'm also fine with just checking for None here for getitem. Let me know which one you prefer

Reply:
I see, I feel both checking are dependent on the behavior of subgraph rewriter, sounds good for now, do we have a longer term solution?

Reply:
Yes that's true, though I think this behavior will stay since the replaced graph may be incorrect if these args are not erased. I don't think there's a longer term solution for now but we can discuss further with the compiler people